serveradmin/settings.example.py

```diff
@@ -189,7 +191,7 @@
             'filters': ['require_debug_false'],
             'level': 'INFO',
             'formatter': 'verbose',
-            'filename': '/var/log/serveradmin.log',
+            'filename': '/tmp/serveradmin.log',
```

Comment:
I would also keep this under `/var/log` if possible.

Reply:
fixed